trunk/NDHMS/HYDRO_drv/module_HYDRO_drv.F

```diff
@@ -948,7 +956,14 @@ subroutine driveChannelRouting(did)
        RT_DOMAIN(did)%QLateral, &
        RT_DOMAIN(did)%HLINK, RT_DOMAIN(did)%ELRT,RT_DOMAIN(did)%CHANLEN,&
        RT_DOMAIN(did)%MannN,RT_DOMAIN(did)%So, RT_DOMAIN(did)%ChSSlp, &
+       !RT_DOMAIN(did)%Bw,RT_DOMAIN(did)%ChannK,&
```

Comment:
Same issue as noted above. Some variables are removed. Also, I added ChannK and qloss below.

Reply:
same comment as above